src/main/java/com/builtbroken/bashrockmakefire/logic/EventHandler.java

```diff
@@ -22,63 +22,52 @@
 {
 
     public static HashMap<BlockPos, ClickData> clickDataHashMap = new HashMap<>();
-    public static List<Block> supportedBlocks = new ArrayList();
-    public static List<IBlockState> supportedBlockStates = new ArrayList();
+    public static List<Block> supportedBlocks = new ArrayList<>();
+    public static List<IBlockState> supportedBlockStates = new ArrayList<>();
```

Comment:
<> is not needed as of java 7 or 8 I think

Reply:
its not *needed*, but IDEA complains :P

Reply:
It shouldn't if you set your code compliance level

Reply:
It will because otherwise its actually an unchecked cast. Empty diamonds just tell it to infer the generic type